frontend/src/tests/components/eMIB/ActionViewTask.test.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { shallow } from "enzyme";
+import LOCALIZE from "../../../text_resources";
+import ActionViewTask from "../../../components/eMIB/ActionViewTask";
+
+describe("renders component's content", () => {
+  const wrapper = shallow(
+    <ActionViewTask
+      taskContent={"Liste of my tasks here..."}
+      reasonsForAction={"Reasons for action here..."}
+    />
+  );
+
+  it("task content", () => {
+    const taskContent = (
+      <div>
+        <p>{LOCALIZE.emibTest.inboxPage.taskContent.task}</p>
+        <p>{"Liste of my tasks here..."}</p>
```

Comment:
To test that the content shows up, you only need to test if line 18 exists. Otherwise if you change the HTML structure this test would break even though the "task content" still appears.

Reply:
Fixed in commit #https://github.com/code-for-canada/project-thundercat/pull/186/commits/8188c7b6c19dee63a8ca51f9b7a4701d606faeac